README.md

```diff
@@ -1 +1,69 @@
 # pin-microservice
+
+ThinBlock's Pin API is built on restify framework. It depends on NodeJS server `v8.11.2` and tests are written with `chai` and runner is `mocha`. The database is MongoDb with mongoose ORM. We use TypeScript langauge to write strongly typed code which has less chances of breaking and better intellisense support in VS Code.
+
+## Installation
+We use `yarn` to install the packages. Do
+```
+ yarn install
+```
+in the project's directory. After installing you need to set up environment variables. In Unix based systems you can do:
+```
+ export TB_PIN_DB_STRING="Mongo db string here"
+```
+
+> Contact the collaborators to get remote db string and node string.
```

Comment:
@abdullahtariq91 Add this remote db string to nuclino's `Database` page. We document there everything

Reply:
Sure. 👍